// chain-registry/testnet/autheotestnet/asset-list.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const info = {
    $schema: '../../assetlist.schema.json',
    chainName: 'autheotestnet',
    assets: [{
            description: 'Native token of Autheo Chain',
            denomUnits: [{
                    denom: 'aauth',
                    exponent: 0
                }, {
                    denom: 'auth',
                    exponent: 6
                }],
            base: 'aauth',
            name: 'Autheo Chain',
            display: 'auth',
            symbol: 'THEO',
            logoURIs: {
                png: 'https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/autheotestnet/images/autheo_logo.png'
            },
            socials: {
                website: 'https://autheo.com'
            },
            typeAsset: 'sdk.coin',
            images: [{
                    png: 'https://raw.githubusercontent.com/cosmos/chain-registry/master/testnets/autheotestnet/images/autheo_logo.png'
                }]
        }]
};
exports.default = info;
