// chain-registry/mainnet/shentu/asset-list.js
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const info = {
    $schema: '../assetlist.schema.json',
    chainName: 'shentu',
    assets: [{
            description: 'The native token of Shentu',
            denomUnits: [{
                    denom: 'uctk',
                    exponent: 0
                }, {
                    denom: 'ctk',
                    exponent: 6
                }],
            base: 'uctk',
            name: 'Shentu',
            display: 'ctk',
            symbol: 'CTK',
            logoURIs: {
                png: 'https://raw.githubusercontent.com/cosmos/chain-registry/master/shentu/images/ctk.png',
                svg: 'https://raw.githubusercontent.com/cosmos/chain-registry/master/shentu/images/ctk.svg'
            },
            coingeckoId: 'certik',
            images: [{
                    png: 'https://raw.githubusercontent.com/cosmos/chain-registry/master/shentu/images/ctk.png',
                    svg: 'https://raw.githubusercontent.com/cosmos/chain-registry/master/shentu/images/ctk.svg'
                }],
            typeAsset: 'sdk.coin'
        }]
};
exports.default = info;
